src/stable/build/commands/compile/wasm_binary/rust/stable_canister_template/src/ic/call_raw.rs

```diff
@@ -89,13 +89,21 @@ fn cleanup(
     global_resolve_id: &str,
     global_reject_id: &str,
 ) -> Result<(), Box<dyn Error>> {
-    let globals = ctx.clone().globals();
-
-    let resolve_callbacks = globals.get::<_, Object>("_azleResolveCallbacks")?;
-    let reject_callbacks = globals.get::<_, Object>("_azleRejectCallbacks")?;
+    dispatch_action(
+        ctx.clone(),
+        "DELETE_AZLE_RESOLVE_CALLBACK",
+        global_resolve_id.into_js(&ctx)?,
+        "azle/src/stable/build/commands/compile/wasm_binary/rust/stable_canister_template/src/ic/call_raw.rs",
```

Comment:
I presume you already looked into generating this path automatically so we don't have to update it in the off chance we move this file?